docs/Apple/release-notes.md

```diff
@@ -6,13 +6,108 @@ hide:
 
 
+## Omnissa IntelligenceSDK for iOS 25.4.0 Release - June 2025
+
+### Minimum Requirements
+
+- Devices running iOS 16.0 or iPadOS 16.0 or newer.
+- WS1SDK version 25.04.0 or newer is required for the two to interact. Note that WS1SDK requires the `fips.xcframework` that is included in its release.
+- not supported:
+   - tvOS devices
+   - app extensions
+   - visionOS for Vision Pro devices
+
+### New Features
+
+- New DEX Telemetry event for iOS: **Critical Battery Level**
+   - This event is triggered when the device reaches the following battery thresholds while the SDK is running:
+      - Reports once when the device reaches a battery level between 16-20%
+      - Reports once when the device reaches a battery level between 11-15%
+      - Reports on every battery level between 0-10%
+
+   When your iOS device is charging, these events will not be sent, even if the device is in the specified range.
+   Device charging will reset the ability to send for all given sections. For example: If we have already sent an event for values between 16-20%, charging followed by discharging will allow another event to be sent for this battery level range.
+
+- New DEX Telemetry attribute for iOS: **Percent Asleep**
+   - Name: percent_asleep
+   - Represents the duration percentage that the device has been asleep since it was booted (on a scale from 0 to 100).
+   
+- New DEX Telemetry attribute for iOS: **Percent Awake**
+   - Name: percent_awake
+   - Represents the duration percentage that the device has been awake since it was booted (on a scale from 0 to 100).
+
+### Resolved Issues
+
+- Creation of the automatic `WS1IntelligenceSDK-Enable-Completed` breadcrumb has been changed to an asynchronous operation to prevent a minor delay in completing the enable process.
+- The IntelSDK now attempts to connect to the `workspaceone.com` domain URLs instead of the previous `vmwservices.com` domain URLs. If the `workspaceone.com` domain URLs are accessible, IntelSDK will construct all its URLs using `workspaceone.com`.
+   - Requirements for allowlisting the necessary URLs can be found here:  [Requirements](https://developer.apple.com/documentation/webkit/wknavigationdelegate/3223382-webview)
+   - For more information on the domain migration, see: [Knowledge Base](https://kb.omnissa.com/s/article/6000882)
+
+- The dSYM upload feature will no longer be supported by the servers after `Jun 30, 2025`
+   - Please remove any build phases that were executing the script which uploaded dSYMs. After Jun 30, 2025, the upload attempt will fail with an error. 
+      - Errors will be prefixed with: `Symbol File upload failed with result`
+   - Additionally, delete the corresponding service account JSON file.
+   - For more information, see our KB article: [End of Availability & Support for the App Crash Symbolication](https://kb.omnissa.com/s/article/6000838)
+
+### Known Issues
+
+none
+
+
+## Omnissa IntelligenceSDK for iOS 25.1.2 Release - February 2025
```

Comment:
Thanks for adding the 25.x.x release notes as well!

Reply:
No problem!